docs/release-notes.md

```diff
@@ -43,6 +43,7 @@ key](https://getfedora.org/security/).
 - Stabilize OpenShift spec 4.14.0, targeting Ignition spec 3.4.0
 - Add OpenShift spec 4.15.0-experimental, targeting Ignition spec
   3.5.0-experimental
+- Support s390x layouts in `boot_device` section (fcos 1.6.0-exp, openshift 4.15.0-exp)
```

Comment:
This should be moved to the "upcoming release"  as this is currently listed in a release which has already been tagged and released.

Reply:
Hi @prestist ,
 Does that mean it should be  add to  ```Upcoming Butane 0.20.0 (unreleased)``` Features?